eng/Signing.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+
+  <PropertyGroup>
+    <!-- This file is also imported in Publish.proj, so condition on the Sign project. -->
+    <EnableDefaultArtifacts Condition="'$(MSBuildProjectName)' == 'Sign'">false</EnableDefaultArtifacts>

```

Comment:
You can also just `<ItemsToSign Remove="@ItemsToSign" />`

Reply:
This handles both signing and post-build signing (admittedly not needed here) and doesn't depend on evaluation order, so I rather use the property here.

https://github.com/dotnet/arcade/blob/22f9fcfafdbf7e7eac90c12078971621aec4a374/src/Microsoft.DotNet.Arcade.Sdk/tools/Sign.props#L27-L36